vpd-tool/include/utils.hpp

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include "types.hpp"
+
+#include <nlohmann/json.hpp>
+#include <sdbusplus/asio/property.hpp>
+#include <sdbusplus/server.hpp>
+
+#include <iostream>
+
+namespace vpd
+{
+namespace utils
```

Comment:
either we go with tool_utils as in tool_constants or we go there also with only constants.
Choose 1.